### PR TITLE
Lastfm commands

### PR DESCRIPTION
**Pull request by metehus**
Last.fm commands
-
Adds new lastfm command to get an user info and search for a track, album or artist on last.fm

### New commands
`s!lastfm user <user>`
`s!lastfm track <query>`
`s!lastfm album <query>`
`s!lastfm artist <query>`

### Images
Track command
Album command
Artist command
User command

### CODE REVIEW

**Comment by Doges** [comment]
o de baixo é corno

File: src/locales/en-US/commands.json [hunk 1]
```diff
@@ -724,5 +724,61 @@
   "support": {
     "commandDescription": "Shows the link to Switchblade support server",
     "clickHere": "**[Click here to join my support server](https://invite.switchblade.xyz)**"
+  },
+  "lastfm": {
+    "commandDescription": "Get an user information or search for a track, album or artist on last.fm",
+    "commandUsage": "<action> <query>",
+    "userHelp": "To get an user's information, use {{- command}} <user>",
```
Comment (pedrofracassi) [inline src/locales/en-US/commands.json]:
```suggestion
    "userHelp": "To get information about an user, use {{- command}} <user>",
```

File: src/locales/en-US/commands.json [hunk 2]
```diff
@@ -724,5 +724,61 @@
...
+    "trackHelp": "To get an track, use {{- command}} <query>",
```
Comment (pedrofracassi) [inline src/locales/en-US/commands.json]:
```suggestion
    "trackHelp": "To get information about a track, use {{- command}} <query>",
```

File: src/locales/en-US/commands.json [hunk 3]
```diff
@@ -724,5 +724,61 @@
...
+    "albumHelp": "To get an album, use {{- command}} <query>",
```
Comment (pedrofracassi) [inline src/locales/en-US/commands.json]:
```suggestion
    "albumHelp": "To get information about an album, use {{- command}} <query>",
```

File: src/locales/en-US/commands.json [hunk 4]
```diff
@@ -724,5 +724,61 @@
...
+    "artistHelp": "To get an artist, use {{- command}} <query>",
```
Comment (pedrofracassi) [inline src/locales/en-US/commands.json]:
```suggestion
    "artistHelp": "To get information about an artist, use {{- command}} <query>",
```

**Comment by metehus** [comment]
@pedrofracassi better now?

**Comment by dpaiv0** [comment]
For numbers, use Intl() so it formats accordingly to the guild's language (see #599, lines 6, 7, 23 and 34)

**Comment by metehus** [comment]
CONVOCO TODOS OS CORNOS PARA FAZEREM REVIEW

**Pull request merged**